packages/node-sdk/paima-utils-backend/src/cde-access-internals.ts

```diff
@@ -80,25 +80,25 @@ export async function internalGetAllOwnedNfts(
 
 export async function internalGetOwnedNfts(
```

Comment:
We'll probably want some other utility functions added around dynamic primitives. For example, if I'm monitoring 100 NFT contracts that I added dynamically, how do I get owners of all of them?

I think probably we just need something like:
1. Some way to know the number of IDs that exists for a dynamic primitive
2. A utility function to create a dynamic primitive cdeName from a base name + an ID

Reply:
> how do I get owners of all of them?

Currently you can use the function I added to get the dynamic primitives by parent, and extract the address from the config. This requires parsing the config though, but we could expose a wrapper otherwise.

> A utility function to create a dynamic primitive cdeName from a base name + an ID

How would you use this though? I'm not sure in which case you would want to use the numeric id, instead of getting the list and storing the names in case you want direct access later.